scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -8,10 +8,10 @@
   tasks:
     - name: OE setup | Set installer URLs from the OE storage account
       set_fact:
-        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.11.100.3.exe"
-        intel_psw_hash:      "72D0C5DCFC24E0DB5B9FB3C2CB5B0242002D1A0DE2A8D4B4AEC862D88C7CBC13"
-        intel_dcap_url:      "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20DCAP%20for%20Windows%20v1.9.100.3.exe"
-        intel_dcap_hash:     "7B09D4455FDFC3DE10B6BC7F4645C16E2E450E41118D0C3E69878078192B4BA4"
+        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.12.100.4.exe"
```

Comment:
Have these files been uploaded to blob storage? I'm not able to download them

Reply:
They are uploaded now, thanks!